mne/viz/ica.py

```diff
@@ -338,8 +345,18 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
     else:
         kind = "Epochs"
 
+    
     epochs_src = ica.get_sources(inst)
-    ica_data = np.swapaxes(epochs_src.get_data()[:, picks, :], 0, 1)
+
+    # if reject is set, reject
+    data = epochs_src.get_data()
+
+    if (reject == 'auto'):
+        reject = None if not hasattr(ica, 'reject_') else ica.reject_ 
```

Comment:
`reject = getattr(ica, "reject", None)`

Reply:
Python is amazing :D

Reply:
I shoud have realized this type of function already existed haha

Reply:
... and that it takes a default argument!